### PR TITLE
Dim pure white text & increase keybind text size in keybind list

### LINKED ISSUE #2417: [ENG-1765] Use a less bright white
### Discussed in [https://github.com/spacedriveapp/spacedrive/discussions/2400](https://github.com/spacedriveapp/spacedrive/discussions/2400)

<div type='discussions-op-text'>

<sup>Originally posted by **ketonik** April 25, 2024</sup>
So using the dark theme, naturally, the font colour appears to be RGB 255,255,255, I rekon tone it down a bit cos it's very bright and almost creates a halo effect around it, like a bright LED, maybe back it off a touch, perhaps RGB 230.230,230?

I think Apple's Finder is RGB 222,222,222</div>

<sub>From [SyncLinear.com](https://synclinear.com) | [ENG-1765](https://linear.app/spacedriveapp/issue/ENG-1765/use-a-less-bright-white)</sub>

### PR DESCRIPTION
**Pull request by PineappleRind**
- Use rgb(235,235,235) for the brightest text color in dark mode (Closes #2417)

- Increase keybind text size in settings for better legibility (before/after):
  before after

Fixes #2417

### CODE REVIEW

**Comment by iLynxcat** [comment]
Thanks for contributing, will do a proper review on this tomorrow!

**Comment by iLynxcat** [comment]
so sorry, I got caught up with some other issues, going to see how this looks locally today & make any needed tweaks so we can merge!

File: packages/ui/style/colors.scss [hunk 1]
```diff
@@ -47,7 +47,7 @@
 	// menu
 	--color-menu: var(--dark-hue), 15%, 10%;
 	--color-menu-line: var(--dark-hue), 15%, 14%;
-	--color-menu-ink: var(--dark-hue), 10%, 100%;
+	--color-menu-ink: var(--dark-hue), 10%, 92%;
```
Comment (iLynxcat) [inline packages/ui/style/colors.scss:50]:
```suggestion
	--color-menu-ink: var(--dark-hue), 25%, 92%;
```

File: packages/ui/style/colors.scss [hunk 2]
```diff
@@ -10,14 +10,14 @@
...
-	--color-ink: var(--dark-hue), 0%, 100%;
+	--color-ink: var(--dark-hue), 0%, 92%;
 	--color-ink-dull: var(--dark-hue), 10%, 70%;
 	--color-ink-faint: var(--dark-hue), 10%, 55%;
 	// sidebar
 	--color-sidebar: var(--dark-hue), 15%, 7%;
 	--color-sidebar-box: var(--dark-hue), 15%, 16%;
 	--color-sidebar-line: var(--dark-hue), 15%, 23%;
-	--color-sidebar-ink: var(--dark-hue), 0%, 100%;
+	--color-sidebar-ink: var(--dark-hue), 0%, 92%;
```
Comment (iLynxcat) [inline packages/ui/style/colors.scss:20]:
```suggestion
	--color-sidebar-ink: var(--dark-hue), 15%, 92%;
```

File: packages/ui/style/colors.scss [hunk 3]
```diff
@@ -10,14 +10,14 @@
 	--color-accent-faint: 208, 100%, 64%;
 	--color-accent-deep: 208, 100%, 47%;
 	// text
-	--color-ink: var(--dark-hue), 0%, 100%;
+	--color-ink: var(--dark-hue), 0%, 92%;
```
Comment (iLynxcat) [inline packages/ui/style/colors.scss:13]:
```suggestion
	--color-ink: var(--dark-hue), 35%, 92%;
```

**Review by myung03 — COMMENTED** [review]
lgtm! thanks so much for contributing :) @iLynxcat is going to take a look at some tint colours as well so I will pass it off to them to approve.

File: interface/app/$libraryId/settings/client/keybindings.tsx [hunk 4]
```diff
@@ -174,7 +174,7 @@ function KeybindTable({ data }: { data: ShortcutCategory['shortcuts'] }) {
 
 					return symbols.map((symbol) => (
 						<div key={symbol} className="inline-flex items-center">
-							<kbd className="ml-2 rounded-lg border border-app-line bg-app-box px-2 py-1 text-[10.5px] tracking-widest shadow">
+							<kbd className="ml-2 rounded-lg border border-app-line bg-app-box px-1.5 py-0.5 text-sm tracking-widest shadow">
```
Comment (iLynxcat) [inline interface/app/$libraryId/settings/client/keybindings.tsx:177]: I like the way this looks. We may revise it in the future as part of an upcoming planned change regarding keyboard shortcuts, but this is a nice inclusion!

**Review by iLynxcat — CHANGES_REQUESTED** [review]
I put some suggested tint changes in here but overall this is nice. I don't have a mini-LED or OLED display to test this on right now, so if we get more complaints we can revise again. Thank you for contributing!

**Review by iLynxcat — APPROVED** [review]

**Pull request merged**